src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -119,7 +120,7 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
   def annotatedMembers() = cpg.annotation
     .code(".*@Value.*")
     .where(_.member)
-    .map { x => (x.parameterAssign.head, x.member.head.name) }
+    .map { x => (x.parameterAssign.head, x.member.head) }
```

Comment:
On line number 121, currently the regex is matched on `code` which is not a safe option instead match it on `fullName` similar to it is been done in `annotatedParameters` function

Reply:
I tried but it does not work with `fullName`

```
joern> cpg.annotation.code(".*@Value.*").l 
res2: List[Annotation] = List(
  Annotation(
    id = 1912L,
    argumentIndex = -1,
    argumentName = None,
    code = "@Value(\"${slack.base.url}\")",
    columnNumber = None,
    fullName = "<unresolvedNamespace>.Value",
    lineNumber = None,
    name = "Value",
    order = 1
  )
)

joern> cpg.annotation.fullName(".*@Value.*").l 
res3: List[Annotation] = List()
```

Reply:
The below works. Any reason why `fullName` is safer?

```
joern> cpg.annotation.fullName(".*Value.*").l 
res4: List[Annotation] = List(
  Annotation(
    id = 1912L,
    argumentIndex = -1,
    argumentName = None,
    code = "@Value(\"${slack.base.url}\")",
    columnNumber = None,
    fullName = "<unresolvedNamespace>.Value",
    lineNumber = None,
    name = "Value",
    order = 1
  )
)

```

Reply:
Done this